src/server/services/list/helpers/build-services-table-data.js

```diff
@@ -1,91 +1,52 @@
-import union from 'lodash/union.js'
-import unionBy from 'lodash/unionBy.js'
-
 import { sortBy } from '~/src/server/common/helpers/sort/sort-by.js'
 import { sortByName } from '~/src/server/common/helpers/sort/sort-by-name.js'
-import { fetchRepositories } from '~/src/server/common/helpers/fetch/fetch-repositories.js'
-import { repositoriesDecorator } from '~/src/server/common/helpers/decorators/repositories.js'
-import { serviceToEntityRow } from '~/src/server/services/list/transformers/service-to-entity-row.js'
-import { fetchDeployableServices } from '~/src/server/common/helpers/fetch/fetch-deployable-services.js'
+import { entityToEntityRow } from '~/src/server/services/list/transformers/entity-to-entity-row.js'
 import { buildSuggestions } from '~/src/server/common/components/autocomplete/helpers/build-suggestions.js'
-import { fetchInProgressFilters } from '~/src/server/services/list/helpers/fetch/fetch-in-progress-filters.js'
-import { fetchInProgressStatus } from '~/src/server/services/helpers/fetch/fetch-in-progress-status.js'
-import { createServiceStatusToService } from '~/src/server/common/transformers/create-service-status-to-service.js'
-import { fetchDeployableServicesFilters } from '~/src/server/services/list/helpers/fetch/fetch-deployable-services-filters.js'
 import { sortByOwner } from '~/src/server/common/helpers/sort/sort-by-owner.js'
+import { fetchEntities } from '~/src/server/common/helpers/fetch/fetch-entities.js'
+import { fetchFilters } from '~/src/server/common/helpers/fetch/fetch-filters.js'
+import { entityOwnerDecorator } from '~/src/server/test-suites/helpers/decorators/entity-owner-decorator.js'
 
 async function buildServicesTableData({
   service,
   teamId,
   isAuthenticated,
   userScopeUUIDs
 }) {
-  const kind = 'microservice'
+  const type = 'Microservice'
```

Comment:
nit: `type` is reserved word in js therefore I generally avoid using it as a const name. Hence `kind`

Reply:
ah. but param is now called type in PBE 😬